packages/address-book/address-book/avax/tokens/tokens.ts

```diff
@@ -15,6 +15,17 @@ const AVAX = {
 } as const;
 
 const _tokens = {
+  'BTC.b': {
```

Comment:
adding the decimal might break the app, can you change the name to BTCb

Reply:
Fixed